Assets/MRTK/Examples/Demos/Audio/Scripts/MicrophoneAmplitudeDemo.cs

```diff
@@ -0,0 +1,239 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Audio;
+using Microsoft.MixedReality.Toolkit.SpatialAwareness;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Examples
+{
+    /// <summary>
+    /// Demonstration class using <see cref="WindowsMicrophoneStream"/> to select the voice microphone and
+    /// adjust the spatial awareness mesh based on the amplitude of the user's voice.
+    /// </summary>
+    [RequireComponent(typeof(AudioSource))]
+    public class MicrophoneAmplitudeDemo : MonoBehaviour
+    {
+#if MICSTREAMDLL_PRESENT
```

Comment:
super-nit, but can we remove `DLL` from the define? makes it a little more readable, and i don't _think_ we've used DLL in past ones (but i could be wrong!)